core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.kt

```diff
@@ -280,7 +280,7 @@ abstract class CoreMainActivity : BaseActivity(), WebViewProvider {
   }
 
   fun openSupportKiwixExternalLink() {
-    externalLinkOpener.openExternalUrl(KIWIX_SUPPORT_URL.toUri().browserIntent())
+    externalLinkOpener.openExternalUrl(KIWIX_SUPPORT_URL.toUri().browserIntent(), false)
```

Comment:
@viditpawar0 We have the same functionality for custom apps as well. So in `CustomMainActivity` we have a function `openExternalUrl`. With this function, we are opening the `support_url` which is the same as the Kiwix app so we should refactor the code there as well.